gdbgui/src/js/SourceCode.jsx

```diff
@@ -165,8 +169,10 @@ class SourceCode extends React.Component {
   ) {
     let row_class = ["srccode"];
 
-    if (is_gdb_paused_on_this_line) {
+    if (is_gdb_paused_on_this_line === 1) {
```

Comment:
This is a boolean, it doesn't make sense to compare to a number. If the semantics around this have changed, the variable name and type should change.